<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QMainWindow" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1171</width>
    <height>465</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1171</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>570</width>
      <height>310</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>570</width>
      <height>310</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>* Дважды кликните по графику, чтобы сохранить его</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>143</y>
      <width>221</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="showGraph2">
       <property name="text">
        <string>Uэталон.(Uвх.)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dotOnGraph">
       <property name="text">
        <string>Точки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="lineOnGraph">
       <property name="text">
        <string>Линии</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>10</y>
      <width>170</width>
      <height>153</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="Search">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Connect">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Соединение</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="FoundedPort"/>
         </item>
         <item>
          <widget class="QPushButton" name="Disconnect">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Разъединение</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>58</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="send">
         <property name="text">
          <string>Отправить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="text">
          <string>Остановить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>11</y>
      <width>238</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="infoChannel_2">
         <property name="text">
          <string>Выберите блок АЦП:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="block_1">
             <property name="text">
              <string>Блок 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="block_2">
             <property name="text">
              <string>Блок 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="block_3">
             <property name="text">
              <string>Блок 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="block_4">
             <property name="text">
              <string>Блок 4</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="infoChannel">
         <property name="text">
          <string>Выберите канал:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="channel_1">
             <property name="text">
              <string>Канал 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="channel_2">
             <property name="text">
              <string>Канал 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="channel_3">
             <property name="text">
              <string>Канал 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="channel_4">
             <property name="text">
              <string>Канал 4</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>86</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>(не более 3.3 В)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>10</y>
      <width>202</width>
      <height>107</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="infoMode">
       <property name="text">
        <string>Режим прохода напряжений:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="choose_mode_1">
       <property name="text">
        <string>от MIN до MAX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="choose_mode_2">
       <property name="text">
        <string>от MAX до MIN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="choose_mode_3">
       <property name="text">
        <string>от MIN до MAX и от MAX до MIN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="choose_mode_4">
       <property name="text">
        <string>RAND ( MIN - MAX )</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>858</x>
      <y>10</y>
      <width>16</width>
      <height>123</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>873</x>
      <y>10</y>
      <width>171</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="infoStep_2">
       <property name="text">
        <string>Установите напряжение:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="infoStep">
           <property name="text">
            <string>Установите шаг, В:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="infoMIN">
           <property name="text">
            <string>Установите MIN, В:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="infoMAX">
           <property name="text">
            <string>Установите MAX, В:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="infoNum">
           <property name="text">
            <string>Кол-во значений:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="step">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="min">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="max">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="num">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
